song_match/cube/note_cube.py

```diff
@@ -35,3 +35,11 @@ def turn_on_light(self) -> None:
         """
         cube_light = self._song.get_cube_light(self._cube.cube_id)
         self._cube.set_lights(cube_light)
+
+    def flash_light_red(self) -> None:
+        """Flash the light on the cube red.
+
+        :return: None
+        """
+        cube_light = self._song.get_cube_light(1)
```

Comment:
Maybe add a TODO comment here so we remember to come back to it in issue #7

Reply:
Indeed.